<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera Image</string>
    </property>
    <widget class="QPushButton" name="pushButton_stopProcess">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_startProcess">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Camera</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_FPS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>FSP: </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_fps">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_SelectCamera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Primary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Secundary</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_selectCamera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select camera device:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_selectCameraResolution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select camera resolution:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_SelectCameraResolution">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>320 x 240</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>640 x 480</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>800 x 600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1024 x 768</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1280 x 720</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1920 x 1080</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_reference">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Reference Image</string>
    </property>
    <widget class="QPushButton" name="pushButton_resetReference">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset Reference Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_setReference">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Reference Image</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_filter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Filter Image</string>
    </property>
    <widget class="QSpinBox" name="spinBox_guassianBlur">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_gaussianBlur">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>- Gaussian blur ratio:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Results</string>
    </property>
    <widget class="QPushButton" name="pushButton_saveResults">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Results</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>190</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
